docs/index.md

```diff
@@ -7,17 +7,31 @@ title: Welcome!
   <img src="images/logo.svg" alt="Logo for NarrateMyWay" width="300">
 </div> -->
 
-[Explain who we are and why we are completing the project]
+<div align="center"> 
+  <h1 class="fw-300">NarrateMyWay</h1>
+</div>
 
-We are a group of 5th Year Electronics & Software Engineering students...
+<div align="center" class="fs-6 fw-300"> 
+  We are a group of 5th Year Electronics & Software Engineering students at the University of Glasgow, completing a project to create a navigation application for the visually impaired. Acting as audio signs giving users a better understanding of their surrounding to improve outdoor and indoor navigation.
```

Comment:
> Acting as audio signs giving users a better understanding of their surrounding to improve outdoor and indoor navigation.

Change to:

Our app provides audio signage, giving users ...

Reply:
Thanks Miklas! Made the change.